source/InputController.cpp

```diff
@@ -212,3 +226,27 @@ void InputController::touchEndedCB(const cugl::TouchEvent& event, bool focus) {
 	// Only need to update the position on touch end event
 	tapLoc.set(event.position);
 }
+
+#pragma mark -
+#pragma mark Click Polling
```

Comment:
Nit: We're not really polling if we're using a callback

Reply:
oops, forgot to change the name to callback again